.github/workflows/security-check.yml

```
@@ -2,25 +2,25 @@ name: Tebra Security Check
       - 'feature/ehr-auth-data-retrieval'
   schedule:
-    - cron: '0 0 * * 1'  # Run weekly on Mondays at midnight
+    - cron: '0 0 * * 1'  # Run weekly on Mondays at midnigh
 
 jobs:
   security-check:
     runs-on: ubuntu-latest
-    
+
     steps:
     - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)